common/changes/@microsoft/rush/watch_2021-01-29-00-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add a new parameter \"--watch\" that interactively watches for filesystem changes and rebuilds the affected Rush projects; this feature can also be used with custom bulk commands (GitHub #2458, #1122)",
```

Comment:
The watching isn't really "interactive," is it? We don't provide an interface for a user to interact with anything in Rush.

Reply:
Removed the word "interactively"